bigquery/tests/test_query_builder.py

```diff
@@ -1086,3 +1087,19 @@ def test_bind_non_string_arguments(self):
             query_statement,
             "SELECT x FROM table WHERE value = True"
         )
+
+    def test_bind_string_arguments_strating_with_single_quote(self):
+        from bigquery.query_builder import ParameterizedQuery
+
+        query = ParameterizedQuery(self.in_statement_input)
+
+        query_statement = query.bind(data_set='table',
+                                     value="'a', 'b'")
+
+        print('\n\nquery_statement\n\n')
+        print(query_statement)
```

Comment:
I guess you have to delete lines 1099 and 1100

Reply:
Whooops